storage/memory/memory.go

```diff
@@ -70,6 +70,26 @@ func (r *Backend) Open(options url.Values) {
 	go r.maintenance()
 }
 
+func (r Backend) GetLastDataItems(tenant string, id string, numOfItems int) ([]storage.DataItem, error) {
+	res := make([]storage.DataItem,numOfItems)
+	r.checkID(tenant, id)
+	t, _ := r.tenant[tenant]
```

Comment:
it's a best practice to check whether the `tenant` exists in the map:
```Go
t, ok := r.tenant[tenant]
if !ok {
  return nil, fmt.Errorf("No such tenant: %s", tenant)
}
```

Reply:
`checkID` checks if the tenant exists.